ansible/roles/cfg-logging/tasks/main.yml

```diff
@@ -7,7 +7,5 @@
 
 - name: Place logging config file
   template: src={{src}} dest={{dest}} owner={{owner}} group={{group}} mode=0644
-  when: "'{{group_name}}' in group_names"
```

Comment:
That's what I figured. We'll need to have a discussion about whether to move forward with having a data container for the configs or not and how we would implement it if so. If it's one container per environment, then we don't really need the guard anymore, but if we do something else we probably will.